studio/src/pages/user_account/AuthComponent.vue

```diff
@@ -574,7 +638,7 @@ export default defineComponent({
     },
     confirmSSOProviderDelete() {
       this.isDeleteSSOProviderDialogOpen = false;
-      appDeleteSSOProvidersById(user.alias)
+      appDeleteSSOProvidersById(this.providerInfo.providerId)
```

Comment:
If this method is to delete SSO provider, we needs `alias` instead of `providerID`. As it is the unique identifier.